wp-parsely.php

```diff
@@ -197,21 +195,21 @@ public function initialize_settings() {
 		// These are the Required Settings.
 		add_settings_section(
 			'required_settings',
-			'Required Settings',
+			__( 'Required Settings', 'wp-parsely' ),
 			array( $this, 'print_required_settings' ),
 			self::MENU_SLUG
 		);
 
 		// Get the API Key.
-		$h = 'Your Site ID is your own site domain ( e.g. `mydomain.com` )';
+		$h = __( 'Your Site ID is your own site domain ( e.g. `mydomain.com` )', 'wp-parsely' );
 
 		$field_args = array(
 			'option_key' => 'apikey',
 			'help_text'  => $h,
 		);
 		add_settings_field(
 			'apikey',
-			'Parse.ly Site ID <div class="help-icons"></div>',
+			__( 'Parse.ly Site ID', 'wp-parsely' ),
```

Comment:
Is the omission of these `<div class="help-icons"></div>` elements intentional? I don't see it detailed in the commit messages.

Reply:
Good call - I meant to mention this on the notes and forgot.

Yes, it is intentional to remove them. WordPress used to have support for showing help icons in wp-admin, but that got removed (not very accessible), so the divs in these labels were redundant.